package.json

```diff
@@ -14,13 +14,12 @@
   "scripts": {
     "build": "yarn run build:source && yarn run build:types",
     "build:clean": "rimraf dist '**/*.tsbuildinfo' && yarn build",
-    "build:docs": "yarn workspaces foreach --parallel --interlaced --verbose run build:docs",
-    "build:source": "yarn workspaces foreach --exclude @metamask/core-monorepo --parallel --interlaced --verbose run build",
+    "build:docs": "yarn workspaces foreach --no-private --parallel --interlaced --verbose run build:docs",
```

Comment:
I see that the test scripts still get run for private packages, which makes sense. Thoughts on leaving it enabled for builds and doc builds as well? Those may both be useful for private packages.

Reply:
Oh wait, no, we check types as part of the `build:types` step. Maybe that's covered then.